src/libs/seedData.ts

```diff
@@ -0,0 +1,12 @@
+import UserRepository from '../repositories/user/UserRepository';
+
+export default function seed() {
```

Comment:
this function should be wrapped in Promise or it will work asynchronously wherever it will be called and I don't think you want that.

Reply:
Done